MediaPlayer/tests/MediaPlayer-specs.js

```diff
@@ -105,4 +115,272 @@ describe('MediaPlayer', () => {
+
+		test('should update `repeat` value of controls frame to `one` when `repeat` button is clicked once', () => {
+			const handleRepeat = jest.fn();
+			// `onRepeat` added for code coverage purposes
+			render(
+				<MediaPlayer data-testid="media-player" onRepeat={handleRepeat}>
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const repeatButton = screen.getByLabelText('Repeat');
+			const controlsFrame = screen.getByTestId('media-player').children.item(3);
+
+			userEvent.click(repeatButton);
+
+			expect(controlsFrame).toHaveAttribute('repeat', 'one');
+		});
+
+		test('should update `repeat` value of controls frame to `all` when `repeat` button is clicked twice', () => {
+			render(
+				<MediaPlayer data-testid="media-player">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const repeatButton = screen.getByLabelText('Repeat');
+			const controlsFrame = screen.getByTestId('media-player').children.item(3);
+
+			userEvent.click(repeatButton);
+			userEvent.click(repeatButton);
+
+			expect(controlsFrame).toHaveAttribute('repeat', 'all');
+		});
+
+		test('should update `repeat` value of controls frame to back to `none` when `repeat` button is clicked 3 times', () => {
+			render(
+				<MediaPlayer data-testid="media-player">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const repeatButton = screen.getByLabelText('Repeat');
+			const controlsFrame = screen.getByTestId('media-player').children.item(3);
+
+			userEvent.click(repeatButton);
+			userEvent.click(repeatButton);
+			userEvent.click(repeatButton);
+
+			expect(controlsFrame).toHaveAttribute('repeat', 'none');
+		});
+
+		test('should have `disabled` attribute when `disabled` is defined', () => {
+			render(
+				<MediaPlayer data-testid="media-player" disabled>
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const mediaSlider = screen.getByRole('slider', {hidden: true});
+			const mediaButtons = screen.getAllByRole('button');
+
+			expect(mediaSlider).toHaveAttribute('disabled');
+			mediaButtons.forEach(button => {
+				expect(button).toHaveAttribute('disabled');
+			});
+		});
+
+		test('should render `currentTime`', () => {
+			render(
+				<MediaPlayer data-testid="media-player">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+
+			const currentTime = screen.getByText('00:00');
+
+			expect(currentTime).toBeInTheDocument();
+		});
+
+		test('should have `paused` class on render', () => {
+			render(
+				<MediaPlayer data-testid="media-player">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFiles[0]} type="audio/mp3" />))}
```

Comment:
Is there a reason you chose audioFile[0] instead of audioFile?

Reply:
It was left like this by mistake. I changed back to `src={audioFiles}` and for the next 3 tests as well